lib/galaxy/tools/parameters/basic.py

```diff
@@ -2103,6 +2201,7 @@ def to_dict(self, trans, other_values=None):
     genomebuild=GenomeBuildParameter,
     select=SelectToolParameter,
     color=ColorToolParameter,
+    select_tag=SelectTagParameter,
```

Comment:
We don't call ``data`` parameters ``select_data`` and this doesn't apply to all tags - so I'm going to switch this to ``group_tag`` - I hope that is okay? I'll add an example tool, test case, and update the XSD. Let me know if there is any issue with this.

Reply:
Yes, that makes more sense!